server/src/main/java/org/candlepin/policy/js/pool/PoolRules.java

```diff
@@ -555,6 +555,21 @@ private void syncBranding(Pool pool, Pool existingPool) {
         return incomingProvided;
     }
 
+    private boolean changedProductsInSet(Set<Product> products,
+        Map<String, Product> changedProducts) {
+
+        if (products != null && changedProducts != null) {
+            for (Product product: products) {
```

Comment:
Checkstyle failure here: need a space between product and the colon. Should also pull the method declaration into one line as well.

Reply:
looks like our checkstyle is fine with these changes. I changed it anyway